From cc2dd7ce6c88891d3387b8d5daeab20e567a7228 Mon Sep 17 00:00:00 2001
From: "Alexander E. Patrakov" <patrakov@gmail.com>
Date: Thu, 10 Apr 2014 21:13:43 +0600
Subject: [PATCH 02/31] Name HDMI outputs uniquely

On Haswell hardware, there are multiple HDMI outputs capable of
digital sound output. As they were identically named, KDE's control
center was unable to distinguish them, restored the wrong profile and
thus routed sound to the wrong HDMI monitor.

Also, having identically-named menu items in other mixer applications
looks like a bug.
---
 src/modules/alsa/mixer/profile-sets/extra-hdmi.conf | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/modules/alsa/mixer/profile-sets/extra-hdmi.conf b/src/modules/alsa/mixer/profile-sets/extra-hdmi.conf
index 96652f8..191a652 100644
--- a/src/modules/alsa/mixer/profile-sets/extra-hdmi.conf
+++ b/src/modules/alsa/mixer/profile-sets/extra-hdmi.conf
@@ -116,7 +116,7 @@ priority = 3
 direction = output
 
 [Mapping hdmi-stereo-extra1]
-description = Digital Stereo (HDMI)
+description = Digital Stereo (HDMI 2)
 device-strings = hdmi:%f,1
 paths-output = hdmi-output-1
 channel-map = left,right
@@ -124,7 +124,7 @@ priority = 2
 direction = output
 
 [Mapping hdmi-surround-extra1]
-description = Digital Surround 5.1 (HDMI)
+description = Digital Surround 5.1 (HDMI 2)
 device-strings = hdmi:%f,1
 paths-output = hdmi-output-1
 channel-map = front-left,front-right,rear-left,rear-right,front-center,lfe
@@ -132,7 +132,7 @@ priority = 1
 direction = output
 
 [Mapping hdmi-stereo-extra2]
-description = Digital Stereo (HDMI)
+description = Digital Stereo (HDMI 3)
 device-strings = hdmi:%f,2
 paths-output = hdmi-output-2
 channel-map = left,right
@@ -140,7 +140,7 @@ priority = 2
 direction = output
 
 [Mapping hdmi-surround-extra2]
-description = Digital Surround 5.1 (HDMI)
+description = Digital Surround 5.1 (HDMI 3)
 device-strings = hdmi:%f,2
 paths-output = hdmi-output-2
 channel-map = front-left,front-right,rear-left,rear-right,front-center,lfe
@@ -148,7 +148,7 @@ priority = 1
 direction = output
 
 [Mapping hdmi-stereo-extra3]
-description = Digital Stereo (HDMI)
+description = Digital Stereo (HDMI 4)
 device-strings = hdmi:%f,3
 paths-output = hdmi-output-3
 channel-map = left,right
@@ -156,7 +156,7 @@ priority = 2
 direction = output
 
 [Mapping hdmi-surround-extra3]
-description = Digital Surround 5.1 (HDMI)
+description = Digital Surround 5.1 (HDMI 4)
 device-strings = hdmi:%f,3
 paths-output = hdmi-output-3
 channel-map = front-left,front-right,rear-left,rear-right,front-center,lfe
-- 
1.9.3

